colour/colorimetry/illuminants.py

```diff
@@ -80,3 +86,42 @@ def D_illuminant_relative_spd(xy):
 
     return SpectralPowerDistribution('CIE Standard Illuminant D Series',
                                      distribution)
+
+
+def CIE_standard_illuminant_A(wl):
```

Comment:
Just tripped over this: Might something like `evaluate_CIE_standard_illuminant_A` be a better name, to indicate that it is a function?
Right now it sounds more like a class to me.